src/build.gradle

```diff
@@ -122,7 +122,7 @@ subprojects {
     task copyTestConfig(type: Copy, dependsOn: ':loadPropertiesTask') {
         from 'src/test/resources'
         include '*'
-        into 'build/classes/test/'
+        into 'out/test/classes/'
         expand(rootProject.ext.props)
```

Comment:
Even after I set the output directories back to the old paths, IntelliJ changed them back when I reloaded the project! But if we're both using the latest IntelliJ may as well hard-code these to the newly correct values in lieu of a proper fix.